pom.xml

```diff
@@ -215,7 +215,8 @@
 				<artifactId>maven-compiler-plugin</artifactId>
 				<version>3.11.0</version>
 				<configuration>
-					<release>8</release>
+					<source>1.8</source>
```

Comment:
I am pretty sure we want to revert this one to use `<release>` instead of `<source>` and `<target>`. If I recall correctly the newer Java compilers (Java 9+)  no longer support separate `source` and `target` flags, and this will break build on all of the more recent compilers. The new compilers use `release`.

Of course, using `release` will break compilation with Java 8. But, if we must break something (and we don't have a choice here), I'd rather not break the more modern versions. Rather, you probably want to start compiling using `javac` from Java 9 or newer...

Reply:
Here is an [article](https://saker.build/blog/javac_source_target_parameters/index.html) that describes the issue better than I can...

Reply:
Are you using JDK 9 now?  I normally use JDK 11 but there are conflicts with the current dependency tree.  I will revert this change.

Reply:
Done.

Reply:
I'm using 11 for the CI (and 17 for myself). 9 is simply the lowest version that works...

Reply:
Excellent.  So we can say that `nom-tam-fits` supports JDK 17?  I'd rather keep using that if so.  Thanks!